torchbenchmark/models/DALLE2_pytorch/__init__.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import random
+import torch
+
+import numpy as np
+
+from torchbenchmark.util.env_check import set_random_seed
+
+from dalle2_pytorch import DALLE2, Unet, Decoder, DiffusionPriorNetwork, DiffusionPrior, OpenAIClipAdapter
+import typing
+
+torch.backends.cudnn.deterministic = False
+torch.backends.cudnn.benchmark = False
+from pathlib import Path
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import COMPUTER_VISION
+
+import io
+
+
+class Model(BenchmarkModel):
+    task = COMPUTER_VISION.GENERATION
+    DEFAULT_TRAIN_BSIZE = 16
+    DEFAULT_EVAL_BSIZE = 16
```

Comment:
Similarly, if there is a default batch size used in the original model code, please add it here.